google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -250,6 +289,16 @@ public void downloadTo(Path path) {
     downloadTo(path, new BlobSourceOption[0]);
   }
 
+  /**
+   * Downloads this blob to the given outputStream.
```

Comment:
If the word output stream conveys the literal meaning, please leave it expanded as "Downloads this blob to the given output stream." like line 245.